docs/generated/galasactl_auth_tokens_get.md

```diff
@@ -13,7 +13,8 @@ galasactl auth tokens get [flags]
 ### Options
 
 ```
-  -h, --help   Displays the options for the 'auth tokens get' command.
+  -h, --help        Displays the options for the 'auth tokens get' command.
+  -i, --id string   An optional flag that is required to return the access tokens of the currently logged in user.The input must be a string
```

Comment:
`-i` doesn't really make sense for a shorthand - so we can remove it.

Also, I think `--loginid` would be clearer since `--id` could be confused with the ID of a token (which is shown when running an `auth tokens get`).

The description can be improved:
1. "an optional flag that is required" is confusing - something like "Optional. ..." would be better
2. The flag allows you to get the tokens for a specific user, not just the currently logged in user
3. Instead of "return the access tokens", perhaps "retrieves the access tokens"
4. Missing a space after the "." in ".The input must be a string"